generator/generator.yml

```
@@ -1144,3 +1144,31 @@ modules:
       yrfUpTime:
         help: The time (in seconds) since the network management portion of the system was last re-initialized - 1.3.6.1.4.1.1182.2.2.4
         scale: 0.01
+
+# MSA fieldserver gateways
+#
+# Gateway configuration guides:
+# * General info: https://cdn.chipkin.com/assets/uploads/2022/Mar/FieldServer_Configuration_Guide_03-19-58-11.pdf
+# * SNMP: see "Protocol Driver Manual - SNMP STD" at https://us.msasafety.com/smc/protocols
+  fieldserver:
+    walk:
+      - "FIELDSERVER-STD-MIB-v1-00aF::analogInputsTable"
+      - "FIELDSERVER-STD-MIB-v1-00aF::analogOutputsTable"
+      - "FIELDSERVER-STD-MIB-v1-00aF::analogValuesTable"
+      - "FIELDSERVER-STD-MIB-v1-00aF::binaryInputsTable"
+      - "FIELDSERVER-STD-MIB-v1-00aF::binaryOutputsTable"
+      - "FIELDSERVER-STD-MIB-v1-00aF::binaryValuesTable"
+    overrides:
+      aiDescription: { type: DisplayString, ignore: true }
```

Please don't mix yaml syntax.

```suggestion
      aiDescription:
        type: DisplayString
        ignore: true
```

---

Sure thing. Done.